### ISSUE TITLE
Weird styles for hidden links

### ISSUE THREAD

**Issue by Igloczek** [SnowdogApps/magento2-alpaca-components#58]
Hidden link IMO is for not visible links created just for accessibility sake, not a definitely visible stuff, with some weird indentation and for some reason positioned absolutely with predefined size... :thinking:

Source:
https://github.com/SnowdogApps/magento2-alpaca-ui-components/blob/develop/components/01-globals/typography/link/_link.scss#L21-L27

**Comment by anqaka** [comment]
I've updated styles. It's for links visible only for screen readers and when they're focused. For a11y (like skip to content link, or some info in complex forms fields).
